packages/source-iottwinmaker/src/data-binding-provider/types.ts

```diff
@@ -0,0 +1,114 @@
+import { Query, TimeSeriesData } from '@iot-app-kit/core';
+
+export interface ITwinMakerEntityDataBindingContext {
```

Comment:
most interfaces here are moved from scene-composer with small modification. the copies in scene composer will be removed in next change